features/rni.feature

```diff
@@ -0,0 +1,202 @@
+    - categorie wordt gevuld met de naam van de categorie waarop de RNI-deelnemer gegevens heeft aangeleverd
+      | categorie | naam           |
+      | 01        | Persoon        |
+      | 04        | Nationaliteit  |
+      | 06        | Overlijden     |
+      | 07        | Inschrijving   |
+      | 08        | Verblijfplaats |
+
+
+    Achtergrond:
+      Gegeven landelijke tabel "RNI-deelnemerstabel" heeft de volgende waarden
+      | code | omschrijving                                      |
+      | 101  | Belastingdienst (inzake heffingen en toeslagen)   |
+      | 201  | Sociale Verzekeringsbank (inzake AOW, Anw en AKW) |
+
+
+    Scenario: RNI-deelnemer op geleverd gegeven
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                            | waarde                                      |
+      | burgerservicenummer             | 000000012                                   |
+      | RNI-deelnemer (01.88.10)        | 101                                         |
+      | Omschrijving verdrag (01.88.20) | Belastingverdrag tussen België en Nederland |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | voornamen (02.10)     | Peter  |
+      | geslachtsnaam (02.40) | Jansen |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                            |
+      | type                | RaadpleegMetBurgerservicenummer   |
+      | burgerservicenummer | 000000012                         |
+      | fields              | naam.voornamen,naam.geslachtsnaam |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam          | waarde |
+      | voornamen     | Peter  |
+      | geslachtsnaam | Jansen |
+      En heeft de persoon een 'rni' met de volgende gegevens
+      | naam                   | waarde                                          |
+      | deelnemer.code         | 101                                             |
+      | deelnemer.omschrijving | Belastingdienst (inzake heffingen en toeslagen) |
+      | omschrijvingVerdrag    | Belastingverdrag tussen België en Nederland     |
+      | categorie              | Persoon                                         |
+
+    Scenario: RNI-deelnemers op geleverde gegevens uit meerdere categorieën
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                            | waarde                                      |
+      | burgerservicenummer             | 000000024                                   |
+      | RNI-deelnemer (01.88.10)        | 101                                         |
+      | Omschrijving verdrag (01.88.20) | Belastingverdrag tussen België en Nederland |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | voornamen (02.10)     | Peter  |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
```

Comment:
'verblijfplaats' ook toevoegen aan 'tableNameMap' in 'step_defs.js':

```
	['verblijfplaats','lo3_pl_verblijfplaats']
```

Reply:
tableNameMap staat nog niet in develop. kan ik alleen doen door [chore/setup-e2e-automation](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/tree/chore/setup-e2e-automation) te mergen in deze PR. Zou betekenen dat bij het mergen van deze RNI PR in develop direct ook de hele chore/setup-e2e-automation meekomt. Hoe regelen we dit?